content/Watsonx/WatsonxAI/ref100.md

```diff
@@ -24,3 +24,27 @@ updated: 2023-10-09
 - **Encoder-only model** – Models with best cost performance trade-off for non-generative use cases but require task-specific labeled data for fine-tuning.
  
 - **Encoder-decoder model**  - Models that support both generative and non-generative use cases. These have the best cost-performance trade-off for generative use cases when the input is large but the generated output is small.
+
+
+## Tokens
+
+A _token_ is a collection of characters that has semantic meaning for a model. Tokenization is the process of converting the words in your prompt into tokens.
+
+Within the **Prompt Lab** text is converted to tokens before being processed by foundation models.
+
+The correlation between words and tokens is complex:
+
+- Sometimes a single word is broken into multiple tokens
+- The same word might be broken into a different number of tokens, depending on context (such as: where the word appears, or surrounding words)
+- Spaces, newline characters, and punctuation are sometimes included in tokens and sometimes not
+- The way words are broken into tokens varies from language to language
+- The way words are broken into tokens varies from model to model
+- For a rough idea, a sentence that has 10 words could be 15 to 20 tokens.
```

Comment:
nitpick: The metric I've seen the most is .75 words per token.  Could we make the example line up with that ratio? 15 words would be around 20 tokens.